course-v2/layouts/partials/quiz_multiple_choice_choice.html

```diff
@@ -1,14 +1,16 @@
 <div class="multiple-choice-div">
-	<label>
-		<input type='radio' name={{ .questionId }} class="multiple-choice-radio">{{.choiceText}}
+	<label class="d-flex align-items-center">
+		<input type='radio' name={{ .questionId }} class="multiple-choice-radio">
+		{{ .choiceText | markdownify }}
```

Comment:
Why not use `.RenderString` here? The `markdownify` function is mostly still part of Hugo as a compatibility measure for older sites, and ultimately it calls `site.Home.RenderString` anyway. The `.RenderString` function is part of the `Page` context, so it does need to be called from a page, but in Hugo 0.111.0 the global `page` func was added. Directly on a page, you can call `.RenderString` because the context (`.`) *is* a `Page`. Here in this partial there is no passed in `Page` context, but you can call the new global `page` function to get that context, and then `.RenderString` will always be called in the context of the current `Page` being rendered. With `markdownify`, it's always run in the context of the "home" page. In this case I don't think that's a problem, but in the Hugo forum thread I posted above people mostly recommend using `.RenderString` wherever you can over `markdownify`. A lot of the code here in `ocw-hugo-themes` uses `markdownify` because a majority of it was written before `.RenderString` was even available. I think going forward, we should use `.RenderString` instead of `markdownify` wherever possible and potentially refactor old code to use it as well.

Reply:
Thanks. In the back of my mind I had a feeling I was missing something - I was actually trying `.RenderString` some days back for Course Collection. But I looked at the codebase, and `markdownify` was being used pretty much everywhere, so I decided to use it.

I have updated the PR with the changes